src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,37 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, sender, body, timeStamp, onToggleLike, liked }) => {
+  const handleLikeClick = () => {onToggleLike(id);
+  };
```

Comment:
I like this pattern of sending just the `id` to `onToggleLike` since it keeps all the state management and message object creation confined to `App`.

The formatting here is a little different than most folks would expect, I would consider dropping `onToggleLike(id);` to its own line or moving up the closing brace:
```js
const handleLikeClick = () => {
    onToggleLike(id);
};
```
Or
```js
const handleLikeClick = () => { onToggleLike(id) };
```